Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -66,12 +71,18 @@ class AdminExtractor implements ExtractorInterface, TranslatorInterface, Securit
      */
     public function __construct(Pool $adminPool, LoggerInterface $logger = null)
     {
+        @trigger_error(
+            'Implementing the Symfony\Component\Translation\TranslatorInterface for '.__CLASS__
+            .' is deprecated since version 3.x and will be removed in 4.0.',
+            E_USER_DEPRECATED
+        );
```

Comment:
I need to talk with you guys about this change, IMO the only task of the AdminExtractor is to extract translations, so is there a further need of this class?

If yes, what would be the solution to handle this?